koinos/protocol/protocol.proto

```diff
@@ -4,6 +4,7 @@ package koinos.protocol;
 option go_package = "github.com/koinos/koinos-proto-golang/koinos/protocol";
 
 import "koinos/options.proto";
+import "koinos/chain/chain.proto";
```

Comment:
I don't think chain needs to be imported.

Reply:
The import is used to access the `chain.object_space` message